x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_to_case_actions.tsx

```diff
@@ -123,6 +131,7 @@ export const useAddToCaseActions = ({
         <EuiContextMenuItem
           aria-label={ariaLabel}
           data-test-subj="add-to-existing-case-action"
+          key="add-to-existing-case-action"
```

Comment:
error in tests pointed out that all members of the context menu did not have keys